src/components/TreeView/TreeView.test.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import {
+  render, screen,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import { FOLIO_RECORD_TYPES } from '../ListTemplate';
+
+import css from '../RecordTypesSelect/RecordTypesSelect.css';
+
+import { TreeView } from './TreeView';
+
+const recordsData = {
+  children: [
+    {
+      itemMeta: FOLIO_RECORD_TYPES.INSTANCE,
+      children: [
+        {
+          itemMeta: FOLIO_RECORD_TYPES.HOLDINGS,
+          children: [{ itemMeta: FOLIO_RECORD_TYPES.ITEM }],
+        },
+        { itemMeta: FOLIO_RECORD_TYPES.MARC_HOLDINGS },
+      ],
+    },
+    { itemMeta: FOLIO_RECORD_TYPES.MARC_BIBLIOGRAPHIC },
+    {
+      itemMeta: {
+        ...FOLIO_RECORD_TYPES.ORDER,
+        captionId: 'ui-data-import.recordTypes.orderLine',
+      },
+    },
+    { itemMeta: FOLIO_RECORD_TYPES.INVOICE },
+    { itemMeta: FOLIO_RECORD_TYPES.MARC_AUTHORITY },
+  ],
+  connections: [],
+};
+
+const renderItemData = item => { return <div>{item.itemMeta.type}</div>; };
+
+const renderTreeView = ({
+  data,
+  indentation,
+  spacing,
+  isLocalLTR,
+  className,
+  renderItem,
+}) => {
+  const component = (
+    <TreeView
+      data={data}
+      indentation={indentation}
+      spacing={spacing}
+      isLocalLTR={isLocalLTR}
+      className={className}
+      renderItem={renderItem}
+    />
+  );
+
+  return render(component);
```

Comment:
If the component renders text anywhere, you need to use **renderWithIntl** and **translationProperties** instead of **render**